recipes/rrbgen/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.0.6" %}
+{% set sha256 = "2cfd00be202a164212949e290459736359e80a064f2992d59f57501629f218ee" %}
+
+package:
+  name: rrbgen
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/rwdavies/rrbgen/releases/download/{{ version }}/rrbgen_{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  script: '$R CMD INSTALL --build --install-tests .'
+
+requirements:
+  build:
+    - r-base >=3.5.0
```

Comment:
This looks wired, have you used the skeleton generator for this?

Reply:
I started with an existing meta.yaml from a different recipe and then modified as seemed appropriate until it passed the tests. I can re-do from a skeleton if this looks too weird

Reply:
That would be nice and please let me know which template you used, maybe we also need to fix this.

Reply:
Unfortunately it seems CRAN package submission is offline until the New Year and this isn't currently in CRAN, so I can't seem to start directly from a skeleton.

I used a combination of https://github.com/bioconda/bioconda-recipes/blob/master/recipes/r-spp/meta.yaml (in the docs, suggested as a recipe for R package not on CRAN) and https://github.com/bioconda/bioconda-recipes/blob/master/recipes/abeona/meta.yaml (written by a friend). On re-review, neither have anything like this in the build section. I think I just added R in every part of the requirements section and it worked so I figured it was OK

Are these tests (the circleci ones) not comprehensive? My thought process when submitting this pull recipe was if anything was wrong, the tests would fail, and I could read the logs / re-submit until I understood what was supposed to go in every section